site/public/js/grade-inquiry.js

```diff
@@ -1,6 +1,25 @@
 /* global buildCourseUrl, WebSocketClient */
 /* exported initGradingInquirySocketClient, onComponentTabClicked, onGradeInquirySubmitClicked, onReady, onReplyTextAreaKeyUp */
 
+function saveDraft() { 
+    //display the saved draft text on grade inquiry box
+    var draftContent = localStorage.getItem('content');
+    if(draftContent === null) {
+        draftContent = {};
+    }else{
+        draftContent = JSON.parse(draftContent);
+    }
+    var elements = document.getElementsByClassName('markdown-textarea fill-available ');
+    for (var i = 0; i < elements.length; i++) {
```

Comment:
instead of using the variable var, please use let or const instead

Reply:
Hello,

I have made some coding style changes to the code. Would you mind taking a moment to review it when you have a chance? Thank you